geps/gep-3793/index.md

```diff
@@ -184,15 +225,353 @@ Gateway controller write a new `parentRefs` stanza to the resource.
+```
+
+A Route MAY include explicit `parentRefs` in addition to setting
+`spec.useDefaultGateway` to `true`. In this case, the Route will be a
+candidate for being bound to default Gateways, but it will also be bound to
+its explicitly-specified `parentRefs`. This allows Ana to create a single
+Route that handles N/S traffic via the default Gateways and also handles E/W
+traffic via a Service, for example.
+
+All other characteristics of a defaulted Route MUST behave the same as if all
+default Gateways were explicitly specified in `parentRefs`.
+
+##### Examples
+
+**Simple N/S Route**: The following HTTPRoute would route _all_ HTTP traffic
+arriving at any default Gateway to `my-service` on port 80:
+
+```yaml
+apiVersion: gateway.networking.k8s.io/v1
+kind: HTTPRoute
+metadata:
+  name: my-route
+spec:
+  useDefaultGateway: true
+  rules:
+  - backendRefs:
+    - name: my-service
+      port: 80
+```
+
+**N/S and E/W Route**: The following HTTPRoute would be bound to both any
+default Gateways and to a Service named `face` in the `faces` namespace,
+permitting a single Route to handle both N/S traffic (via the default Gateway)
+and E/W traffic (via the Service):
+
+```yaml
+apiVersion: gateway.networking.k8s.io/v1
+kind: HTTPRoute
+metadata:
+  name: ns-ew-route
+spec:
+  useDefaultGateway: true
+  parentRefs:
+  - kind: Service
+    name: face
+    namespace: faces
+  rules:
+  - backendRefs:
+    - name: face
+      port: 80
+```
+
+**Multiple Gateways**: A defaulted Route MAY include both a `DefaultGateway`
+and other specific named Gateways, although this is not expected to be common
+in practice:
+
+```yaml
+apiVersion: gateway.networking.k8s.io/v1
+kind: HTTPRoute
+metadata:
+  name: multi-gateway-route
+spec:
+  useDefaultGateway: true
+  parentRefs:
+  - kind: Gateway
+    name: my-gateway
+    namespace: default
+  rules:
+  - backendRefs:
+    - name: my-service
+      port: 80
+```
+
+##### `status` for a Defaulted Route
+
+When a defaulted Route is claimed by a default Gateway, the Gateway MUST use
+`status.parents` to announce that it has claimed the Route, for example:
+
+```yaml
+status:
+  parents:
+  - name: my-default-gateway
+    namespace: default
+    controllerName: gateway.networking.k8s.io/some-gateway-controller
+    conditions:
+    - type: Accepted
+      status: "True"
+      lastTransitionTime: "2025-10-01T12:00:00Z"
+      message: "Route is bound to default Gateway"
+```
+
+##### Other Considerations
+
+A default Gateway MUST NOT modify the `parentRefs` of a defaulted Route to
+indicate that the Route has been claimed by a default Gateway. This becomes
+important if the set of default Gateways changes, or (in some situations) if
+GitOps tools are in play.
+
+If there are no default Gateways in the cluster, `spec.useDefaultGateway` MUST
+be treated as if it were set to `false` in all Routes, parallel to the
+situation where a Route specifies a Gateway by name, but no Gateway of that
+name exists in the cluster.
```

Comment:
This feels confusing to me. I think you're saying that if `useDefaultGateway` is true, and there are no default Gateways in the cluster, the controller should just ignore it? Saying that it should treat `useDefaultGateway` as false doesn't feel correct even though the result would be the same.